src/guide/testing-the-jdk.md

```diff
@@ -48,13 +48,15 @@ public class TestXY {
 
 This example only utilizes three jtreg specific tags, `@test`, `@summary`, and `@run`. `@test` simply tells jtreg that this class is a test, and `@summary` provides a description of the test. `@run` tells jtreg how to execute the test. In this case we simply tell jtreg to execute the main method of the class `TestXY`. `@run` isn't strictly necessary for jtreg to execute the test, an implicit `@run` tag will be added if none exists. However, for clarity and in order to avoid bugs it's recommended to always explicitly use the `@run` tag.
 
-There are several other tags that can be used in jtreg tests. You can for instance associate the test with a specific bug that this test is a regression test for.
+There are several other tags that can be used in jtreg tests. You can for instance associate the test with a specific bug that this test is a regression test for. The bug id is written without the `JDK-` prefix.
 
 ~~~
-@bug 7000001
+@bug 8272373
 ~~~
 
-Or you can specify a number of requirements that must be fulfilled for jtreg to execute the test.
+You can add several bug ids in the same `@bug` tag, separated by a single space. The bug ids listed here should refer to bugs for which this test is verifying the fix. JBS issues that track changes to the test itself are not listed here.
```

Comment:
Is the comma necessary before “separated”?

Reply:
That's a good question. I'll defer to someone who is native English speaker to determine that.

Reply:
In think it is ok, as it emphasizes the first part of the statement. A slighly more verbose form would be:
You can add several bug ids in the same `@bug` tag, each bug id is separated by a single space.
This in turn could be structured as two separate sentences. 
You can add several bug ids in the same `@bug` tag. Each bug id is separated by a single space.